README.md

```diff
@@ -8,28 +8,29 @@ This is a utility to flash firmware blobs to tenstorrent devices, currently only
 
 ## Getting started
 
-### To Build from git:
+### To Build from GitHub:
```

Comment:
I'd like to keep this generic to 'git' vs. GitHub

Reply:
Good idea. I changed it back to git. Maybe it might make more sense to say "Build from Source"?